src/localization/locales/en_US/translations.json

```diff
@@ -581,6 +581,24 @@
       },
       "total": "Total"
     },
+    "setup": {
+      "email": "Email address",
+      "firstname": "First name",
+      "gender": {
+        "label": "Gender",
+        "options": {
+          "female": "Female",
+          "male": "Male",
+          "other": "Other"
+
+        },
+        "placeholder": "Gender"
+      },
+      "lastname": "Last name",
+      "password": "Password",
+      "setupButton": "Setup",
+      "title": "Setup"
+    },
```

Comment:
I have a feeling some of these translations are also used in other places, so it would be nice if we can reuse them. Note that you only want to reuse translations when they literally point to the same thing, so you can probably reuse stuff from the login/reset password screen and the user props.

Page-specific translations (like _Setup_) you of course do not want to reuse.

Reply:
Do you mean, use the translation of other pages or extract the common parts into their own place?

Reply:
I think it is fine to reuse some translations from other pages, like the profile page. I would not make these translations generic/common for PP, but I think redefining is worse.